web/js/feedback.js

```diff
@@ -30,4 +29,4 @@ export const feedbackModal = (function () {
   };
 
   return self;
-})();
+})({});
```

Comment:
Why did you move `self` from being defined inside the function to being passed in as a parameter?

Reply:
Discussed in slack chat; export constant does not work in this case. We can look at changing this to a named function when we do a review. Util functions are setup in a similar manner as this.

Reply:
Does a naming the function change how parameters are passed in? Before, you could call this function with no parameters, `feedbackModal()`, and it would work fine. Now we have to call the function with an empty object for it to work, `feedbackModal({})`. Why can't we do `var self = {}` inside the function like it was before?

Reply:
Good call, I have made this change and tested it.